core/src/main/scala/filodb.core/memstore/IndexBootstrapper.scala

```diff
@@ -25,13 +25,12 @@ class IndexBootstrapper(colStore: ColumnStore) {
     * @param assignPartId the function to invoke to get the partitionId to be used to populate the index record
     * @return number of updated records
     */
-  def bootstrapIndex(index: PartKeyLuceneIndex,
-                     shardNum: Int,
-                     ref: DatasetRef)
-                     (assignPartId: PartKeyRecord => Int): Task[Long] = {
+  def bootstrapIndexRaw(index: PartKeyLuceneIndex,
```

Comment:
Do we really have to rename this? `raw` does not seem to be a correct keyword here.

Reply:
Raw was used since it is used for raw data cluster. The name change was intended to indicate the purpose of each method.

Reply:
Since raw was not the only other usecase after DS, i asked that question. By default a cluster can be used for anything: for example; to store rules, to store aggregated data, for raw data.